dashboard/app/models/peer_review.rb

```diff
@@ -37,10 +37,8 @@ class PeerReview < ActiveRecord::Base
   belongs_to :level
   belongs_to :level_source
 
-  after_update :mark_user_level, if: :status_changed?
+  after_update :mark_user_level, if: proc {|review| review.status_changed? || review.data_changed?}
```

Comment:
nit: let's be consistent. Why `proc` here and stabby lambda on line 50 below. Also we don't need the param, since this proc is in the context of this class. This can be shortened to
```
proc {status_changed? || data_changed?}
```
or
```
-> {status_changed? || data_changed?}
```

I just ran a quick experiment, and dumped the result of `review == self` in this block and it was always true regardless of whether I defined it as a proc or lambda.